components/infobox/wikis/wildrift/infobox_league_custom.lua

```diff
@@ -70,6 +68,7 @@ end
 function League:defineCustomPageVariables()
 	Variables.varDefine('tournament_patch', _args.patch)
 	Variables.varDefine('tournament_endpatch', _args.epatch)
+	Variables.varDefine('tournament_device', _args.device)
```

Comment:
what is the purpose of this?
will it be accessed by some templates/modules on the pages?
if not kick it